Examples/flp2epn-distributed/src/EPNReceiver.cxx

```diff
@@ -148,6 +144,12 @@ void EPNReceiver::Run()
             LOG(ERROR) << "Could not send acknowledgement without blocking";
           }
         }
+        else
```

Comment:
This is to avoid sending when nothing is connected, as it is in the test mode. Does it break your setup?